mmv1/products/containerattached/Cluster.yaml

```diff
@@ -360,6 +360,7 @@ properties:
     description: |
       Enable/Disable Security Posture API features for the cluster.
     default_from_api: true
+    deprecation_message: '`security_posture_config` is deprecated and will be removed in a future release.'
```

Comment:
Will this field be replaced by a new field or just be removed?

Reply:
It will be removed.